### PR TITLE
Created issue templates

### PR DESCRIPTION
**Pull request by maxlovic**
Adding issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ assignees: ''
 ---
 
 When asking a question please also include where you looked for an answer (so we can update the documentation if needed).
+General questions regarding SRT are encouraged to be asked in the SRT Alliance slack channel: [join](https://slackin-srtalliance.azurewebsites.net/).
```
Comment (stevomatthews) [inline .github/ISSUE_TEMPLATE/question.md]: You are encouraged to ask general questions regarding SRT in the [SRT Alliance Slack channel](https://slackin-srtalliance.azurewebsites.net/).

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+2. Run '....'
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (stevomatthews) [inline .github/ISSUE_TEMPLATE/bug_report.md]: **Desktop (please provide the following information):**

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: "[FR]"
+labels: 'Type: Enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (stevomatthews) [inline .github/ISSUE_TEMPLATE/feature_request.md]: A clear and concise description of what the problem is. For example: "When I try to do X, I am blocked by Y..."

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (stevomatthews) [inline .github/ISSUE_TEMPLATE/feature_request.md]: A clear and concise description of how you would like to see this feature implemented.

**Comment by maxlovic** [comment]
@stevomatthews Applied your corrections.

**Pull request merged**